.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -70,3 +70,17 @@ This change added tests and can be verified as follows:
 
   - Does this pull request introduce a new feature? (yes / no)
   - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
+
+----
+
+# Review Progress
+
+**NOTE: THE REVIEW PROGRESS MUST ONLY BE UPDATED BY AN APACHE FLINK COMMITTER!**
+
+* [ ] 1. The contribution is well-described.
+* [ ] 2. There is consensus that the contribution should go into to Flink.
+* [ ] 3. [Does not need specific attention | Needs specific attention for X | Has attention for X by Y]
```

Comment:
About [3], If the committer identifies [Needs specific attention for X], what do we need to do with PR?
From the points of my view, the committer who change the review process, have to make sure all the specific attention parts are solved.So the [3] should be described : [ ] 3. All the specific attention parts are well-solved.
 What do you think?

Reply:
I think it is fine as it is. When the PR needs special attention, it should be marked at the point 3.. 

A committer who is familiar with the code should then resolve the remaining points 4. and 5, architecture and implementation.

Reply:
Yes, I think in theory, the number of people who should have a look at the PR can be extended. IMO, the committer who finally merges the PR should check that everybody mentioned there commented on the PR or ask them before merging.

Reply:
Make sense to me. +1 to merged.